cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -321,358 +325,252 @@ describe('Credentials', () => {
       cy.clickButton(/^Cancel/);
     });
   });
+});
 
-  describe('Create Credentials of different types', () => {
-    type MockCredentialData = {
-      name: string;
-      required?: Array<{ field: string; dataCy: string }>;
-      fields?: Array<{ id: string; value: string; dataCy: string }>;
-    };
-
-    // FLAKY_06_13_2024
-    it('can create credentials with 30 different credential types', function () {
```

Comment:
Why did we remove this? I don't see it accounted for anywhere else

Reply:
This one originally in #2601.

But, if we do end up removing it, it might make sense to also remove `cypress/fixtures/credentialsTestData.json` which is only used by this test.